octoprint_mrbeam/static/js/app/view-models/user-notification.js

```diff
@@ -156,7 +156,7 @@ $(function () {
                     ),
                     self._getPlaceHolderParameters(
                         gettext("https://support.mr-beam.org/en") +
-                            "/support/solutions/articles/43000703002"
+                            "/support/solutions/articles/43000708776-interlock-malfunction-check-the-lid-switch"
```

Comment:
I think it should be like this instead
```suggestion
                            "/support/solutions/articles/43000708776"
```

Reply:
This is a completely different article and it was a request to change it